themes/bootstrap3/templates/search/filters.phtml

```diff
@@ -128,18 +128,21 @@
   ?>
   <?php // Normal view ?>
   <div class="active-filters hidden-xs">
+    <span class="sr-only"><?=$this->transEscAttr('page_reload_on_deselect_hint') ?></span>
     <?php if ($resetLink && $options->getRetainFilterSetting()): ?>
       <a class="reset-filters-btn" href="<?=$resetLink?>"><?=$this->transEsc('reset_filters_button')?></a>
     <?php elseif ($advancedSearch): ?>
       <p class="adv_search_filters"><?=$this->transEsc('adv_search_filters')?>:</p>
     <?php endif; ?>
     <div class="filters">
+      <span class="sr-only"><?=$this->transEscAttr('applied_filters') ?>:</span>
```

Comment:
The comment you pointed out as outdated was actually talking about this template line and the associated translation string. Sorry for confusing matters by talking about it in entirely the wrong context! :-)

Reply:
Ah, ok. I get it now. Sorry. Committed the change